setup.py

```diff
@@ -25,7 +25,7 @@
         "Topic :: Scientific/Engineering :: Atmospheric Science",
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
-        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.11",
```

Comment:
This is not a published package; I suggest to remove all classifiers at is has no function.

Reply:
@frenck - just removed them all; let me know if you had something different in mind here.